src/Microsoft.Azure.WebJobs.Host/Executors/JobHostConfigurationExtensions.cs

```diff
@@ -238,7 +238,7 @@ public static ServiceProviderWrapper CreateStaticServices(this JobHostConfigurat
 
                 if (functionExecutor == null)
                 {
-                    functionExecutor = new FunctionExecutor(functionInstanceLogger, functionOutputLogger, exceptionHandler, trace, functionEventCollector, loggerFactory);
+                    functionExecutor = new FunctionExecutor(functionInstanceLogger, functionOutputLogger, exceptionHandler, trace, host, config, functionEventCollector, loggerFactory);
```

Comment:
We had removed the new public property we added (JobHost.Configuration) to avoid controversy :) If you like the new property we could add it back - I think it makes sense. Just removed it because it was new surface area that wasn't strictly needed.